src/test/ui/lint/lint-temporary-cstring-as-param.rs

```diff
@@ -0,0 +1,10 @@
+// ignore-tidy-linelength
```

Comment:
Maybe add a comment that this isn't technically incorrect, but it's not immediately obvious?